tests/acceptance/fixtures.py

```diff
@@ -450,12 +450,12 @@ def prepared_test_build(prepared_test_build_base):
 
 
 @pytest.fixture(autouse=True)
-def min_mender_version(request, bitbake_variables):
-    version_mark = request.node.get_closest_marker("min_mender_version")
+def min_mender_client_version(request, bitbake_variables):
+    version_mark = request.node.get_closest_marker("min_mender_client_version")
     if version_mark is None:
         pytest.fail(
             (
-                '%s must be marked with @pytest.mark.min_mender_version("<VERSION>") to '
+                '%s must be marked with @pytest.mark.min_mender_client_version("<VERSION>") to '
                 + "indicate lowest Mender version for which the test will work."
             )
             % str(request.node)
```

Comment:
Once you are on it, modify line 465 below to read `mender-client` instead of mender. For exactly this, no extra modification in `mender-convert` is needed, that I can see.